EIPS/eip-3534.md

```diff
@@ -0,0 +1,318 @@
+
+
+### EIP-2930 Inheritance
+The [EIP-2930 Optional Access List Type Transaction](https://eips.ethereum.org/EIPS/eip-2930) is used as an assumed "base" transaction type for this proposal. 
+However, this is NOT a conceptual dependency; the included `accessList` portion of this proposal (the only differential from post-EIP-155 legacy transaction fields) can readily be removed. 
+Standing on the shoulders of EIP-2930 is only intended to support and further the adoption of next-generation transactions.
+
+### Signature target
+
+The signature signs over the transaction type as well as the transaction data.
+This is done to ensure that the transaction cannot be “re-interpreted” as a transaction of a different type.
+
+## Backwards Compatibility
+
+This EIP requires a hard fork.
+
+## Test Cases
+
+| Segment ID | Block Number | Canonical Block Hash |
+| --- | --- | --- |
+| `e78b1ec3` | `0` | `0xe78b1ec31bcb535548ce4b6ef384deccad1e7dc599817b65ab5124eeaaee3e58` |
+| `01e78b1ec3` | `1` | `0xe78b1ec31bcb535548ce4b6ef384deccad1e7dc599817b65ab5124eeaaee3e58` |
+| `e4e1c0e78b1ec3` | `15_000_000` | `0xe78b1ec31bcb535548ce4b6ef384deccad1e7dc599817b65ab5124eeaaee3e58` |
+| `e8d4a50fffe78b1ec3` | `999_999_999_999` | `0xe78b1ec31bcb535548ce4b6ef384deccad1e7dc599817b65ab5124eeaaee3e58` |
+| `7fffffffffffffffe78b1ec3` | `9223372036854775807` | `0xe78b1ec31bcb535548ce4b6ef384deccad1e7dc599817b65ab5124eeaaee3e58` |
+
+Further test cases, TODO.
+
+## Security Considerations
+
+#### Why 4 bytes of a block hash is "safe enough" for the `ancestorId`
+
+__TL;DR__: The chance of an ineffectual `ancestorId` is about 1 in between ~4 billion and ~40 billion (the greater chance for intentional duplication scenarios, eg. malicious reorgs).
+
+__If a collision _does_ happen__, that means the transaction will be valid on both segments (as is the case under the status quo).
+
+Four bytes, instead of the whole hash (32 bytes), was chosen only to reduce the amount of information required to cross the wire to implement this value.
+Using the whole hash would result in a "perfectly safe" implementation, and every additional byte reduces the chance of collision exponentially.
+
+The goal of the `ancestorId` is to disambiguate one chain segment from another, and in doing so, enable a transaction to define with adequate precision which chain it needs to be on.
+When a transaction's `ancestorId` references a block, we want to be pretty sure that that reference won't get confused with a different block than the one the author of the transaction had in mind.
+
+Block hashes are effectively unique (at least ethereum/go-ethereum assumes so in their database schema), and are [cited as approximations for random numbers in the yellow paper](https://github.com/ethereum/yellowpaper/blob/41c1837f7b1ddcd65f135763c8e965146cd0ab70/Paper.tex#L1261).
```

Comment:
No external links.  Also, this particular citation doesn't seem to add value to this discussion so I recommend just removing it.  We can assume the readers of this document understand the uniqueness and collision resistant properties of the keccak256 hashing algorithm and the block hash selection process.